generators/client-2/files.js

```diff
@@ -140,6 +140,8 @@ const files = {
                 'layouts/profiles/_profile.service.ts',
                 'layouts/profiles/_profile-info.model.ts',
                 'layouts/profiles/_page-ribbon.component.ts',
+                'layouts/main/_jhipster-main.component.ts',
```

Comment:
Added a main component which acts as the main layout. contains the top level  uiviews